src/main/java/com/booking/app/controller/RoomController.java

```diff
@@ -0,0 +1,25 @@
+package com.booking.app.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class RoomController extends BaseController {
+    private static final String API_NAME = "rooms";
+
+    @PostMapping(RoomController.API_NAME + "/{id}")
+    public ResponseEntity<String> createRoom(@RequestBody long id) {
```

Comment:
This should be a room creation.
What you do here is basically sending a room id intro the controller. If so, than how do you know the room capacity, the hotel and other details? This should be a RoomDTO not and it
Also never pass id's in a creation. The id needs to be autogenerated